### DISCUSSION TITLE
Training on simulation and testing on real-world benchmark

### DISCUSSION THREAD

**Discussion by nikola310** [SysCV/qd-3dt#7] [Ideas]
Thank you for sharing your interesting work. Have you perhaps tried training your model on GTA data set and testing on real-world images? It would be interesting to see how model trained on synthetic data responds to real-world data.

**Comment by eborboihuc** [comment]
Thanks for your interest in our work! 
We focus more on combining quasi-dense similarity learning and 3D instance dynamics, to track moving objects in a 3D world. But, we also notice the potential of using our GTA dataset as a data source to transfer the ability on real-world data. It's a great topic to discuss with others if anyone happens to have some experiments about it.
